.gitignore

```diff
@@ -36,4 +36,7 @@ temporal_tls_key.key
 
 # Contract Keys
 mint.json
-authority.json
\ No newline at end of file
+authority.json
+
+#target
+/target
```

Comment:
we need to include this at least for now since the types are depending on it and everyone needs it, might be better to turn it to a build process that fills this folder up

Reply:
@mehdidehdar what are your thoughts on this??